test.yml

```diff
@@ -7,17 +7,27 @@
 
   roles:
     - role: role_under_test
+      when: ansible_distribution_release == 'xenial' or (ansible_os_family == 'RedHat' and ansible_distribution_major_version == '7')
 
     - role: role_under_test
+      openio_repository_openstack_release: queens
       openio_repository_products:
         sds:
-          release: "16.10"
-      when: ansible_distribution != 'Debian'
+          release: '18.04'
+      when: ansible_distribution_release == 'bionic'
 
+
+##    - role: role_under_test
```

Comment:
Ditto, why is that commented code here ? will it be activated in the future, or is this just sample code (a comment would be welcome then) ?